scripts/sign_verify_file_ssh.sh

```diff
@@ -132,20 +140,16 @@ elif [ "$MODE" == "verify" ]; then
     echo "Verifying the signature against allowed signers..."
 
     # Iterate through each principal in the allowed signers file
-    while IFS= read -r line || [[ -n "$line" ]]; do
-        [[ -z "$line" || "$line" == \#* ]] && continue
-
-        # Extract and process each principal
-        principals=$(echo "$line" | cut -d' ' -f1)
-        IFS=',' read -ra principal_list <<< "$principals"
-
-        for principal in "${principal_list[@]}"; do
-            echo "Checking principal: $principal"
-            if ssh-keygen -Y verify -f "$ALLOWED_SIGNERS_FILE" -n file -I "$principal" -s "$SIG_FILE" < "$FILE_TO_SIGN"; then
-                echo "Signature is valid for principal: $principal"
-                exit 0
-            fi
-        done
+    while read -r principal options key
```

Comment:
In general, full format support for the applied signers file is overly complicated for our use case (https://github.com/jelmer/dulwich/issues/1431#issuecomment-2492080239). I do think we need some kind of sanity check on the file (or lines in the file) to ensure we are getting the simplified format that we expect (single simple) for  principal and the optional namespace.

Reply:
Maybe we can do that sanity check later?